packages/@sanity/field/src/diff/components/ChangeTitleSegment.tsx

```diff
@@ -1,9 +1,20 @@
 import * as React from 'react'
+import {Box, rem, Text} from '@sanity/ui'
+import styled from 'styled-components'
 import {FromToIndex, Annotation, FieldChangeNode} from '../../types'
 import {getAnnotationAtPath} from '../annotations'
 import {DiffCard} from './DiffCard'
 
-import styles from './ChangeTitleSegment.css'
+const RoundedCard = styled.div`
+  border-radius: ${({theme}) => rem(theme.sanity.radius[2])};
```

Comment:
Could this have been solved with `<Card radius={2} padding={1}` instead?

Reply:
Wasn't able to get it to work as expected because this component is passed down to the `DiffCard` component :/

Reply:
Looks like this is now not needed because of the changes to the DiffCard component! 🎉